src/libstd/collections/hash/raw/bitmask.rs

```diff
@@ -35,35 +41,36 @@ impl BitMask {
         if self.0 == 0 {
             None
         } else {
-            Some(self.trailing_zeros())
+            Some(unsafe { self.lowest_set_bit_nonzero() })
```

Comment:
just squeezed in a quick codegen fix?

Reply:
Oh yea, I got that one mixed into the iterator changes. It's fine though, so I'll leave it in.